### PR TITLE
docs: CTC meeting minutes 2017-02-01

### PR DESCRIPTION
**Pull request by joshgav**
cc @nodejs/ctc

### CODE REVIEW

File: meetings/2017-02-01.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+Should we unlock `assert` for API changes?
+
+Perhaps not, but we should still support new ES constructs within existing API.
+
+@mylesborins will open an issue to re-evaluate what “locked” policy means,
+becuase JS language changes happen more frequently now.
+
+**Next steps** 
+
+* @Trott to draft proposal based on this discussion as PR.
```
Comment (Trott) [inline meetings/2017-02-01.md]: My ideas of the absolute minimum of what we ought to do here are in https://github.com/nodejs/node/pull/11113 (and perhaps also https://github.com/nodejs/node/pull/11112 although that's not really a content change). While I wouldn't necessarily object to bigger changes, this is what I consider more-or-less the minimum we can do to signal that it's OK to use `assert` as long as you don't need features that aren't there and that we are interested in fixing bugs in `assert` even if they don't affect Node.js core tests. Fixing bugs: yes. New features: no, but there's lots of great options in npm.

**Review by Trott — APPROVED** [review]

File: meetings/2017-02-01.md (hunk 1 above)
Comment (joyeecheung) [reply meetings/2017-02-01.md]: Will we ever put ES.next features in our other APIs? If so, it's very likely that we are gonna need some new features from `assert` for our internal testing code in the furture anyway.

**Review by jasnell — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by evanlucas — APPROVED** [review]

**Comment by joshgav** [comment]
Landed in https://github.com/nodejs/CTC/commit/fa33091646e89e3155c147dbbd532bb6bc5e162d. Thanks for the reviews!

**Pull request merged**